plugins/cisco_umbrella_enforcement/komand_cisco_umbrella_enforcement/actions/add_event/action.py

```diff
@@ -14,31 +14,30 @@ def __init__(self):
         )
 
     def run(self, params={}):
-        domains = params.get("events")
-        events = []
 
-        for event in domains:
-            dstUrl = event.get("dstURL")
-            event["dstUrl"] = dstUrl
-            del event["dstURL"]
-
-            ID = event.get("ID")
-            event["deviceId"] = ID
-            del event["ID"]
-
-            events.append(event)
-
-        try:
-            dictIds = self.connection.api.add_event(events)
-        except Exception:
-            self.logger.error("AddEvent: run: Problem with request")
-            raise Exception("AddEvent: run: Problem with request")
-
-        ids = []
-        for key, value in dictIds.items():
-            ids.append(value)
-
-        return {"ID": ids}
-
-    def test(self):
```

Comment:
Should this test function still be here or are the tests covered under unit tests?

Reply:
Tests are covered by unit tests, we don't really use test functions within the actions anymore.